integration/kubernetes/init.sh

```diff
@@ -164,19 +164,19 @@ configure_network() {
 			info "Restart the CRI-O service due to $issue"
 			sudo systemctl restart crio
 		fi
-		local list_pods="kubectl get -n kube-system --selector app=flannel pods"
+		local list_pods="kubectl get -n kube-flannel --selector app=flannel pods"
```

Comment:
Hey @ryansavino, was testing this out to fix some of my own problems (https://github.com/kata-containers/tests/issues/5400) and I'm wondering if you also need to update kube-system on line 73?